manifest.json

```
@@ -122,6 +122,10 @@
                       "description": "Property values must be a multiple of this number. Only relevant for numeric properties.",
                       "type": "number"
                     },
+                    "enum": {
+                      "description": "Property values must be one of these options, given as a JSON Array. Only relevant for string properties.",
+                      "type": "string"
+                    },
```

I'm curious, why not using an array?

```suggestion
                    "enum": {
                      "description": "Property values must be one of these options, given as a JSON Array. Only relevant for string properties.",
                      "type": "array",
                      "items": {
                        "type": "string"
                      }
                    },
```

---

> I added this option as a string type, where one has to enter a valid JSON array (if no valid JSON is entered, not enum will be added to the property). I thought this may be helpful since one can just copy and paste the array from somewhere. But I think it may be more user-friendly to change the type to array of string?

:D